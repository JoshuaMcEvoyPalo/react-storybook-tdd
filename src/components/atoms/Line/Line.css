.line {
    border: 1px solid #DFE0E2;
    width: 100%;
}