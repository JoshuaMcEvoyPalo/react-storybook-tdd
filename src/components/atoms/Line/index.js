import React from 'react'
import './Line.css'

const Line = () => {
  return (
    <div data-testid="line" className="line" />
  )
}

export default Line
