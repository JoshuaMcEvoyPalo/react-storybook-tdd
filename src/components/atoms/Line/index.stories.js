import Line from '.'

export default {
  title: 'Atoms / Line',
  component: Line
}

const Template = (args) => <Line {...args} />

export const Default = Template.bind({})
Default.args = {}