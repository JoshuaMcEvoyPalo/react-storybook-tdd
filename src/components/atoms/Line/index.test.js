import renderer from 'react-test-renderer'
import Line from '.'

describe('Line tests', () => {
  it('Should render Line correctly', () => {
    const tree = renderer.create(<Line />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})