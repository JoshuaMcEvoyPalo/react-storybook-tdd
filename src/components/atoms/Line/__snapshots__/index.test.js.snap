// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Line tests Should render Line correctly 1`] = `
<div
  className="line"
  data-testid="line"
/>
`;
